### PR TITLE
Feature/99 add version

### LINKED ISSUE #99: Add current project version to element
In order to quickly determine which version is being used, add a hidden element with a data-version property with a value that equals the current version of the product.

For example:
<span class="hidden" data-version="2.0.1"></span>

The value should automatically update whenever a release is made.

### PR DESCRIPTION
**Pull request by frontendherodk**
Fixes #99

### CODE REVIEW

File: src/app/navigation.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="scn">
+<div class="scn" data-version="{{$ctrl.version}}">
```
Comment (dami-gg) [inline src/app/navigation.html]: Please add {{::$ctrl.version}} here.

Comment (frontendherodk) [inline src/app/navigation.html]: Good point!

**Comment by dami-gg** [comment]
:+1:

**Comment by frontendherodk** [comment]
👍

**Pull request merged**